shark-graph/src/main/java/shark/internal/aosp/ByteArrayTimSort.kt

```diff
@@ -54,7 +54,7 @@ fixed number of bytes N. Each entry is compared by a custom comparator.
  * sort, assuming the input array is large enough to warrant the full-blown
  * TimSort. Small arrays are sorted in place, using a binary insertion sort.
  */
-@Suppress("ComplexMethod", "ComplexCondition", "LongMethod", "LongParameterList")
+@Suppress("detekt.complexity", "detekt.style")
```

Comment:
Had to bump version to use this new cool feature - suppressing entire rulesets 😃